### PR TITLE
Fix credo issues

### PR DESCRIPTION
**Pull request by mbramson**
Fix various credo issues

### CODE REVIEW

File: lib/pairmotron/authentication.ex [hunk 1]
```diff
@@ -1,4 +1,16 @@
 defmodule Pairmotron.Authentication do
+  @moduledoc """
+  Helper module for Pairmotron authnetication. Sole responsibility is returning
```
Comment (ericworkman) [inline lib/pairmotron/authentication.ex]: Typo `authnetication` 😿

File: web/models/group.ex [hunk 2]
```diff
@@ -1,4 +1,14 @@
 defmodule Pairmotron.Group do
+  @moduledoc """
+  Groups contain users and allow certain aspects of Pairmotron to be siloed on
+  a per group basis.
+
+  Groups have their own set of pairs, and Users can be in pairs of multiple
+  Groups at the same time. Groups also have their own set of projects.
+
+  Groups also have their own configuration options for how users are paired.
+  (Not yet implemented)
```
Comment (ericworkman) [inline web/models/group.ex:10]: Hah!

File: web/models/user_pair.ex [hunk 3]
```diff
@@ -1,4 +1,8 @@
 defmodule Pairmotron.UserPair do
+  @moduledoc """
+  A UserPair joins together Users and Pairss and is used to represent a
```
Comment (ericworkman) [inline web/models/user_pair.ex]: 🐍 `Pairss`

File: web/views/view_helper.ex [hunk 4]
```diff
@@ -1,10 +1,17 @@
 defmodule Pairmotron.ViewHelpers do
+  @moduledoc """
+  Contains various functions that are used by a number of views/templates. Auto
+  imported into views and templates through web.es.
```
Comment (ericworkman) [inline web/views/view_helper.ex]: ex

**Review by ericworkman — COMMENTED** [review]
Awesome job! I'm super excited about having the @specs available. Just found a couple silly typos.

File: web/models/group.ex (hunk 2 above)
Comment (mbramson) [reply web/models/group.ex:10]: Ya wasn't actually sure if I should even put that sentence there

**Pull request merged**